data/json/furniture_and_terrain/furniture-regional-pseudo.json

```
@@ -3,7 +3,7 @@
     "type": "furniture",
     "id": "f_region_flower",
     "name": "this should never actually show up, it's a pseudo furniture",
-    "description": "this should never actually show up, it's a pseudo furniture",
+    "description": "this should never actually show up, it's a pseudo furniture, seeing this is a bug, please report it.",
```

```suggestion
    "description": "This is pseudo-furniture and should never actually show up. Please report this bug.",
```
You fixed abuse of commas with more comma abuse :3